CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 2.10.1  (2023-04-03)
+### Changed
+- Fix log example file path in unix tarballs
```

Comment:
did you do this? was it done in a previous commit?

Reply:
I did it here: #107